operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1603,33 +1602,34 @@ memcachedExporter:
       drop: [ALL]
     allowPrivilegeEscalation: false
 
-admin-cache:
-  # -- Specifies whether admin-cache should be enabled
-  enabled: true
-
-  # Cache type to be used (options: inmemory, memcahed, redis)
-  cacheType: inmemory
+# -- Cache for admin bucket.
+# If this is disabled, in-memory cache will be set by default.
+# You can use Redis too for cache and set the configuration via structuredConfig.
+# See GEM documentation for Redis configuration option.
+memcached-admin-cache:
```

Comment:
we don't have the memcached prefix on other values in this file - e.g. `index-cache`, `chunks-cache`, `results-cache`. What do you think about doing the same here - just `admin-cache`?

Reply:
also - i just remembered that all the GEM values are at the bottom of this massive values.yaml file. Do you mind moving this section?

https://github.com/grafana/mimir/blob/c7ad9bc469b49e7b078096ac9c569546cc4a5df4/operations/helm/charts/mimir-distributed/values.yaml#L2840-L2841

Reply:
Initially I added memcached prefix so that it is clear the cache is using memcache and user can use another external cache. But I agree with you we should just consistent with other cache block naming in the values. yaml.

I also moved the section to the GEM section.